modules/orchestrator/proc-enabling-orchestrator-plugins-components.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-enabling-orchestrator-plugins-components.adoc_{context}"]
+= Enabling Orchestrator plugins components
+
+{product} includes the following Orchestrator plugin components in the `dynamic-plugins.default.yaml` file, but they are disabled by default:
+
+* `backstage-plugin-orchestrator`
+* `backstage-plugin-orchestrator-backend-dynamic`
+* `backstage-plugin-scaffolder-backend-module-orchestrator-dynamic`
+* `backstage-plugin-orchestrator-form-widgets`
+
+.Procedure
+* Locate your {product-short} configuration and set the `plugins.disabled` parameter to `false` for each required plugin package.
++
+For example, you can enable the core Orchestrator frontend plugin as shown in the following example:
++
+[source,yaml,subs="+attributes,+quotes"]
+----
+plugins:
+  - package: "@redhat/backstage-plugin-orchestrator@{product-chart-version}"
+    disabled: false
+----
```

Comment:
If the all plugins are required, then provide the definitive content rather than a partial example:

```suggestion
+
[source,yaml,subs="+attributes,+quotes"]
----
plugins:
  - package: "@redhat/backstage-plugin-orchestrator@{product-chart-version}"
    disabled: false
  - package: "@redhat/backstage-plugin-orchestrator-backend-dynamic@{product-chart-version}"
    disabled: false
  - package: "@redhat/backstage-plugin-scaffolder-backend-module-orchestrator-dynamic@{product-chart-version}"
    disabled: false
  - package: "@redhat/backstage-plugin-orchestrator-form-widgets@{product-chart-version}"
    disabled: false
----
```

Reply:
Else, consider a 4 steps procedure:

1. Enable the Orchestrator frontend plugin.
2. Enable the Orchestrator backend plugin.
3. Enable the <scaffolder plugin description> plugin.
4. Enable the <forms widget plugin description> plugin.